FirelandsProject/firelands-cata
# Update (Anticheat): BG Fine Tune\Jump False hit

### Changes proposed

- Fine Tune BG AV for horde.

![image](https://user-images.githubusercontent.com/16887899/202926880-368cf520-d75d-47b4-9548-4849d656f1c3.png)

- Fixes False Hit for Advanced Jump Detection for the Celestial Planetarium Observer Battle that has a narrow path that false flags.

### Issues addressed


- Closes

### SOURCE:


- Anticheat Collaboration

### Tests performed


- Tested and no longer any false hits
- 

### How to Test the Changes:


1. Start AV BG as horde. Run to the left wooden frame of the gate before BG Starts
2. Observe no false flags from anticheat.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
